pkg/pgtune/parallel.go

```diff
@@ -31,13 +31,14 @@ var ParallelKeys = []string{
 
 // ParallelRecommender gives recommendations for ParallelKeys based on system resources.
 type ParallelRecommender struct {
-	cpus int
+	cpus         int
+	MaxBGWorkers int
 }
 
 // NewParallelRecommender returns a ParallelRecommender that recommends based on
 // the given number of cpus.
-func NewParallelRecommender(cpus int) *ParallelRecommender {
-	return &ParallelRecommender{cpus}
+func NewParallelRecommender(cpus int, MaxBGWorkers int) *ParallelRecommender {
```

Comment:
`cpus int, MaxBGWorkers int` -> `cpus, maxBGWorkers int`

Reply:
I made the change , except for flag parsing. Go absolutely requires the first letter to be capital.

Reply:
Yes it looks like you figured it out. The flags one is fine to leave exported (and needed), the others were not. Good stuff!